### PR TITLE
Update secrets template and hub for how to drive provisioning

### PR DESCRIPTION
**Pull request by beekhof**

### CODE REVIEW

File: values-secret.yaml.template [hunk 1]
```diff
@@ -3,4 +3,25 @@ secrets:
...
-    secret: PLAINTEXT
+    secret: VALUE
+
+  # Required for automated spoke deployment
+  aws:
+      access_key_id: VALUE
+      secret_access_key: VALUE
+
+# Required for automated spoke deployment
+# DO NOT use ~/ , it will appear to work be the secrets will be empty
```
Comment (mbaldessari) [inline values-secret.yaml.template:14]: This has been fixed via https://github.com/hybrid-cloud-patterns/common/commit/8f03fa648d300adec315e0fb3cdb4afc4d734014 so let's remove this line?

File: values-secret.yaml.template [hunk 2]
```diff
@@ -3,4 +3,25 @@ secrets:
...
+files:
+  # ssh-rsa AAA...
+  publickey: ~/.ssh/id_rsa.pub
```
Comment (mbaldessari) [inline values-secret.yaml.template:17]: I think we should leave them commented out? Won't 'load-secrets' explode if the files do not exist?

**Review by mbaldessari — COMMENTED** [review]
If you want I can fix those nits later on top

File: values-secret.yaml.template (hunk 1 above)
Comment (beekhof) [reply values-secret.yaml.template:14]:
```suggestion
```

**Pull request merged**